runtime/caml/backtrace.h

```diff
@@ -52,7 +62,8 @@
  *   OCaml values of algebraic data-type [Printexc.backtrace_slot]
  */
  /* [Caml_state->backtrace_active] is non zero iff backtraces are recorded.
- * This variable must be changed with [caml_record_backtrace].
+ * This variable must be changed with [caml_record_backtrace] in OCaml or
+ * caml_record_backtraces in C.
```

Comment:
For consistency: `[caml_record_backtraces]` ? or even `[caml_record_backtraces()]` ?

Reply:
Done